src/System.Diagnostics.Process/tests/RemotelyInvokable.cs

```diff
@@ -87,6 +87,18 @@ public static int ReadLine()
             return SuccessExitCode;
         }
 
+        public static string WriteLineReadLineUapCmd()
```

Comment:
this is called on uap by reflection - see https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.Process/tests/ProcessTestBase.Uap.cs#L56 for details. We need to use cmd.exe on uap or otherwise we can't get a process handle